client/rpc-api/src/child_state/mod.rs

```diff
@@ -68,4 +69,13 @@ pub trait ChildStateApi<Hash> {
 		key: StorageKey,
 		hash: Option<Hash>
 	) -> FutureResult<Option<u64>>;
+
+	/// Returns proof of storage for child key entries at a specific block's state.
+	#[rpc(name = "state_getChildReadProof")]
```

Comment:
@mustermeiszer is there a specific reason this RPC is named `state_` and not `childstate_` like the other RPCs in this file?

Reply:
TBH, it's a copy paste mistake from my side, as I grapped the logic from the 'state' impl. So renaming this to 'childstate' would probably be favorable.

Reply:
Ok, cool. I've changed and added an alias to `state_getChildReadProof`. :)